doc/pmempool/pmempool-sync.1.md

```diff
@@ -75,6 +75,26 @@ be opened are recreated.=e=)
 
 ##### Available options: #####
 
+`-b, --bad-blocks`
+
+: Fix bad blocks - it requires creating or reading special recovery files.
+When bad blocks are detected, special recovery files have to created
+in order to fix them safely. A separate recovery file is created per each part
+containing bad blocks. The recovery file is created in the same directory
+where the pool's part is located and with the name
+\<pool-part-file-name\>_badblocks.txt where \<pool-part-file-name\> is the name
+of the pool's part file. These recovery files are automatically removed
+if the sync operation finishes successfully.
+If the last sync operation was interrupted and not finished correctly
+(eg. the application crashed) and bad blocks were being fixed in the meantime,
```

Comment:
I don't like the "were being fixed in the meantime" part. Maybe:
`and bad block fixing procedure was in-flight`? but that doesn't sound great either...